File: python/A 강조하기.py
# def solution(myString):
#     arr = []
#     for elem in myString:
#         if elem == 'a':
#             arr.append('A')
#         elif elem !='A' and elem.upper():
#             arr.append(elem.lower())
#         else:
#             arr.append(elem)
#     return ''.join(arr)

def solution(myString):
    return myString.lower().replace('a', 'A')

File: js/A 강조하기.js
// function soluiton(myString) {
//     return ([...myString].map(v => v.toLowerCase())).map(v => v === 'a' ? v.toUpperCase() : v).join('');
// }


function solution(myString) {
    return myString.toLowerCase().replaceAll('a', 'A');
}